js/react_native/package.json

```diff
@@ -77,8 +77,7 @@
   },
   "dependencies": {
     "@expo/config-plugins": "^7.2.4",
-    "buffer": "^6.0.3",
-    "onnxruntime-common": "file:../common"
```

Comment:
I think "onnxruntime-common" is still a valid dependency. Is there a reason why removing it?

Reply:
It's unnecessary change, I'll revert it.